test/integration/ddl_sharding_key_test.lua

```diff
@@ -179,3 +484,69 @@ pgroup.test_non_unique_index = function(g)
     t.assert_not_equals(result, nil)
     t.assert_equals(#result.rows, 2)
 end
+
+pgroup.test_get_secondary_idx = function(g)
+    local tuple = {4, box.NULL, 'Leo', 44}
+
+    -- insert tuple
+    local result, err = g.cluster.main_server.net_box:call('crud.insert', {
+        'customers_secondary_idx_name_key', tuple
+    })
+
+    t.assert_equals(err, nil)
+    t.assert_not_equals(result, nil)
+    t.assert_equals(#result.rows, 1)
+
+    -- get
+    local result, err = g.cluster.main_server.net_box:call('crud.get', {'customers_secondary_idx_name_key', {'Leo'}})
```

Comment:
Cite from README (https://github.com/tarantool/crud/#get):

> * `key` (`any`) - primary key value

`'Leo'` is not suitable to be a key for `unsigned` primary index. I don't understand the idea of the `test_get_secondary_idx` test.

Reply:
The same for `test_update_secondary_idx` and `test_delete_secondary_idx`.

Reply:
Fixed.

```diff
--- a/test/integration/ddl_sharding_key_test.lua
+++ b/test/integration/ddl_sharding_key_test.lua
@@ -498,7 +498,8 @@ pgroup.test_get_secondary_idx = function(g)
     t.assert_equals(#result.rows, 1)
 
     -- get
-    local result, err = g.cluster.main_server.net_box:call('crud.get', {'customers_secondary_idx_name_key', {'Leo'}})
+    local result, err = g.cluster.main_server.net_box:call('crud.get',
+        {'customers_secondary_idx_name_key', {4, 'Leo'}})
 
     t.assert_str_contains(err.err,
         "Sharding key for space \"customers_secondary_idx_name_key\" is missed in primary index, specify bucket_id")
@@ -522,7 +523,7 @@ pgroup.test_update_secondary_idx = function(g)
     }
 
     local result, err = g.cluster.main_server.net_box:call('crud.update', {
-        'customers_secondary_idx_name_key', {'Victor'}, update_operations,
+        'customers_secondary_idx_name_key', {6, 'Victor'}, update_operations,
     })
 
     t.assert_str_contains(err.err,
@@ -543,7 +544,7 @@ pgroup.test_delete_secondary_idx = function(g)
     t.assert_equals(#result.rows, 1)
 
     local result, err = g.cluster.main_server.net_box:call('crud.delete', {
-        'customers_secondary_idx_name_key', {'Alexander'}
+        'customers_secondary_idx_name_key', {8, 'Alexander'}
     })
 
     t.assert_str_contains(err.err,
```